test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -250,12 +264,34 @@ func (e *Steps) retrieveProfile(id, name, issuerURL, supportedVCContexts,
 	return nil
 }
 
-func (e *Steps) walletConnect(issuerID string) error {
+func (e *Steps) walletConnect(profileID string) error {
 	state := uuid.New().String()
-	e.states[issuerID] = state
+	e.states[profileID] = state
+
+	resp, err := bddutil.HTTPDo(http.MethodGet, //nolint: bodyclose
```

Comment:
Is this a false positive for the linter? If so, you can add a small comment next to the nolint to let other developers know. (I've also encountered this false positive, there's an open issue for it)

`//nolint: bodyclose // False positive`